core/routemgmt/common/apigw-utils.js

```diff
@@ -577,15 +577,15 @@ function generateCliApiFromGwApi(gwApi) {
  *  result - an array of strings.
  *           result[0] : Entire URL
  *           result[1] : protocol (i.e. https)
- *           result[2] : host (i.e. myco.com, 1.2.3.4)
+ *           result[2] : host (i.e. myco.com, 1.2.3.4, myco.com/mywhisk)
  *           result[3] : namespace
  *           result[4] : package name
  *           result[5] : action name
  *           result[6] : action response type (i.e http, json, text, html, or svg)
  */
 function parseActionUrl(actionUrl) {
   console.log('parseActionUrl: parsing action url: '+actionUrl);
-  var actionUrlPattern = /(\w+):\/\/([:\w.\-]+)\/api\/v\d\/web\/([@\w .\-]+)\/([@\w .\-]+)\/([@\w .\-\/]+)\.(\w+)/;
+  var actionUrlPattern = /(\w+):\/\/([:/\w.\-]+)\/api\/v\d\/web\/([@\w .\-]+)\/([@\w .\-]+)\/([@\w .\-\/]+)\.(\w+)/;
```

Comment:
need to escape that new `/` character?

Reply:
it seems to work without escaping the / (because it is inside a [] ??), but  I can test with the / being escaped.   I'm not an expert of the grammar for JS regex...